### PR TITLE
M1 e2e public fixes

### PR DESCRIPTION
**Pull request by TawneeOwl**
## What does this pull request do?

Adds arguments to the dockerfile to allow it to be built different on M1 E2E tests

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [ ] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Review by BenMillar-CGI — DISMISSED** [review]
Looks good to me, let's be careful when pushing the changes through and ensure the end-to-end tests work as expected on staging before pushing further 👍

**Review by said-moj — APPROVED** [review]
👍 LGTM

File: docs/installation.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Next, create the environment and start it up:
     cd cla_public
     virtualenv env --prompt=\(cla_public\)
 
-    source env/bin/activate
```
Comment (psweeting1) [inline docs/installation.md:21]: Why are you removing this? It's handy to have.

Comment (TawneeOwl) [reply docs/installation.md:21]: It shouldn't be removing that, will amend

**Review by Pheonnexx — APPROVED** [review]
LGTM

**Pull request merged**